src/test/java/com/hpe/adm/octane/ideplugins/integrationtests/IntegrationTestBase.java

```diff
@@ -288,5 +300,28 @@ public void createUSer() {
 
     }
 
+    public void createNewUser(){
+        EntityModel userEntityModel = new EntityModel();
+        Set<FieldModel> fields = new HashSet<>();
+        FieldModel typeField = new StringFieldModel("type","user");
+        FieldModel firstName = new StringFieldModel("first_name","john");
+        FieldModel lastName = new StringFieldModel("last_name","doe");
+        FieldModel email = new StringFieldModel("email","john.doe@hpe.com");
+        FieldModel fullname = new StringFieldModel("full_name","John Doe");
+        fields.add(typeField);
+        fields.add(lastName);
+        fields.add(firstName);
+        fields.add(email);
+        fields.add(fullname);
+        userEntityModel.setValues(fields);
```

Comment:
Can be further simplified, you don't need the field value.
You could directly do userEntityModel.setValue(new StringFieldModel("type","user"))
2x less code